tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarClusterSpec.java

```diff
@@ -69,6 +69,15 @@
      *
      * @return the flag whether to enable/disable container log.
      */
+    @Default
```

Comment:
why is this defaulting to false? without container logs any failures are impossible to debug.

Reply:
that's stdout and stderr container streams, we log cluster log files separately and it cannot be disabled